dev/openchpl_preload.sql

```diff
@@ -1302,14 +1302,12 @@ INSERT INTO openchpl.muu_accurate_as_of_date (accurate_as_of_date, last_modified
 INSERT INTO user_permission (user_permission_id, "name", description, authority, last_modified_user) VALUES
 (-2, 'ADMIN', 'This permission confers administrative privileges to its owner.', 'ROLE_ADMIN', -1),
 (1, 'USER_CREATOR' ,'This permission allows a user to create other users', 'ROLE_USER_CREATOR' , -1),
-(2, 'ACB_ADMIN' ,'This permission gives a user write access to their ACBs.', 'ROLE_ACB_ADMIN' , -1),
-(3, 'ACB_STAFF' ,'This permission gives a user read access to their ACBs', 'ROLE_ACB_STAFF' , -1),
-(4, 'ATL_ADMIN' ,'This permission gives a user write access to their ATLs.', 'ROLE_ATL_ADMIN' , -1),
-(5, 'ATL_STAFF' ,'This permission gives a user write access to their ATLs.', 'ROLE_ATL_STAFF' , -1),
+(2, 'ACB' ,'This permission gives a user write access to their ACBs.', 'ROLE_ACB_ADMIN' , -1),
+(4, 'ATL' ,'This permission gives a user write access to their ATLs.', 'ROLE_ATL_ADMIN' , -1),
```

Comment:
Shouldn't the last item be `ROLE_ACB` and `ROLE_ATL`? Without the `_ADMIN` parts?